drape/gl_functions.cpp

```diff
@@ -243,38 +243,23 @@ void GLFunctions::Init(dp::ApiVersion apiVersion)
   s_inited = true;
 
 /// VAO
-#if defined(OMIM_OS_MAC)
+#if !defined(OMIM_OS_WINDOWS)
   if (CurrentApiVersion == dp::ApiVersion::OpenGLES2)
   {
+  #if defined(OMIM_OS_MAC)
+
     glGenVertexArraysFn = &glGenVertexArraysAPPLE;
     glBindVertexArrayFn = &glBindVertexArrayAPPLE;
     glDeleteVertexArrayFn = &glDeleteVertexArraysAPPLE;
     glMapBufferFn = &::glMapBuffer;
     glUnmapBufferFn = &::glUnmapBuffer;
-  }
-  else if (CurrentApiVersion == dp::ApiVersion::OpenGLES3)
-  {
-    glGenVertexArraysFn = &::glGenVertexArrays;
-    glBindVertexArrayFn = &::glBindVertexArray;
-    glDeleteVertexArrayFn = &::glDeleteVertexArrays;
-    glUnmapBufferFn = &::glUnmapBuffer;
-    glMapBufferRangeFn = &::glMapBufferRange;
-    glFlushMappedBufferRangeFn = &::glFlushMappedBufferRange;
-    glGetStringiFn = &::glGetStringi;
-  }
-  else
-  {
-    ASSERT(false, ("Unknown Graphics API"));
-  }
-#elif defined(OMIM_OS_LINUX)
-  glGenVertexArraysFn = &::glGenVertexArrays;
-  glBindVertexArrayFn = &::glBindVertexArray;
-  glDeleteVertexArrayFn = &::glDeleteVertexArrays;
-  glMapBufferFn = &::glMapBuffer;
-  glUnmapBufferFn = &::glUnmapBuffer;
-#elif defined(OMIM_OS_ANDROID)
-  if (CurrentApiVersion == dp::ApiVersion::OpenGLES2)
-  {
+
+  #elif defined(OMIM_OS_LINUX)
+
+    ASSERT(false, ("OpenGLES2 is not yet supported"));
```

Comment:
Hm, but the old code with hardcoded apiOpenGLES3 = false worked ok for a bunch of Linux (Ubuntu) users ..
Or I miss something here?

Reply:
Please see my response in the qt main conversation.

Reply:
Ok, please replace with CHECK(false, (...)).